pkg/cmd/server/api/types.go

```diff
@@ -998,3 +1001,26 @@ type AdmissionConfig struct {
 	// on the master. Order is significant. If empty, a default list of plugins is used.
 	PluginOrderOverride []string
 }
+
+// ClusterResourceOverrideConfig is the configuration for the ClusterResourceOverride
+// admission controller which overrides user-provided container request/limit values.
+type ClusterResourceOverrideConfig struct {
```

Comment:
Not really quota-related per se. It's for ops to tweak overcommit mostly.

Clayton and I had a brief discussion at https://github.com/openshift/origin/pull/6901#discussion-diff-52099169 - I would be happy for something that's more descriptive. Maybe one of your epic half-page names is in order :)